server/liveblog/client_modules/__init__.py

```diff
@@ -13,6 +15,18 @@ def init_app(app):
     service = ClientPostsService(endpoint_name, backend=superdesk.get_backend())
     ClientPostsResource(endpoint_name, app=app, service=service)
 
+    endpoint_name = 'client_collections'
```

Comment:
Should this not be 'client_advertisement_collections'? Just 'client_collections' is very generic and doesn't really describe what it is being represented.